src/modules/vmount/input.cpp

```diff
@@ -60,6 +60,27 @@ int InputBase::update(unsigned int timeout_ms, ControlData **control_data, bool
 		return 0;
 	}
 
+	if (_failsafe_action == 1 && !_failsafe_triggered && _actuator_armed_sub.updated()) {
```

Comment:
Could you please introduce an ENUM for the failsafe action and use that here?

Reply:
Done.